### ISSUE TITLE
Pressing delete produces ? on Linux

### ISSUE THREAD

**Issue by jusw85** [garglk/garglk#469]
Pressing delete produces ? on Linux.
Compiled using 
```
cmake .. -DWITH_TTS=false -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_INSTALL_LIBEXECDIR=lib
```
It appears as far back as a9de2d4641f47c35a98a5b5d2f9788095c856358, but I'm not sure if it appears prior to that.

https://user-images.githubusercontent.com/3406652/124634465-f4f55680-deb8-11eb-8904-14d325ca4702.mp4

It appears for all the ulx and blorb files that I've tested.

**Comment by cspiegel** [comment]
This should be fixed in #471.

Thanks for the report!

**Comment by cspiegel** [comment]
#471 has been merged
